procurement_for_service/models/sale_order.py

```diff
@@ -0,0 +1,43 @@
+# -*- coding: utf-8 -*-
+##############################################################################
+# For copyright and license notices, see __openerp__.py file in root directory
+##############################################################################
+from openerp import models, api, exceptions, _
+
+
+class SaleOrder(models.Model):
+    _inherit = 'sale.order'
+
+    @api.one
+    def action_button_confirm(self):
+        procurement_obj = self.env['procurement.order']
+        res = super(SaleOrder, self).action_button_confirm()
+        for line in self.order_line:
+            routes = line.product_id.route_ids.filtered(
+                lambda r: r.name in ('Make To Order', 'Buy'))
+            if line.product_id.type == 'service' and len(routes) == 2:
```

Comment:
Esto se hace porque el producto tiene que estar definido como 'Make To Order', y como 'Buy'